policy/engine.go

```diff
@@ -33,16 +33,34 @@ type Engine struct {
 }
 
 // Load returns an Engine after loading all of the specified policies.
-func Load(ctx context.Context, policyPaths []string) (*Engine, error) {
+func Load(ctx context.Context, policyPaths []string, capabilities string) (*Engine, error) {
 	policies, err := loader.AllRegos(policyPaths)
 	if err != nil {
 		return nil, fmt.Errorf("load: %w", err)
 	} else if len(policies.Modules) == 0 {
 		return nil, fmt.Errorf("no policies found in %v", policyPaths)
 	}
 
+	// set capabilities
+	var c *ast.Capabilities
+	// if capabilities are not provided as a cmd flag,
+	// then ast.CapabilitiesForThisVersion must be called
+	if capabilities != "" {
+		f, err := os.Open(capabilities)
+		if err != nil {
+			return nil, fmt.Errorf("file couldn't be opened: %w", err)
+		}
+		defer f.Close()
+		c, err = ast.LoadCapabilitiesJSON(f)
+		if err != nil {
+			return nil, fmt.Errorf("capabilities not loaded: %w", err)
+		}
+	} else {
+		c = ast.CapabilitiesForThisVersion()
+	}
```

Comment:
nit: A more idiomatic approach would be to initialize `c` to `ast.CapabilitiesForThisVersion()` then then override that if the `capabilities` string is not empty.

Reply:
indeed, although this is now in LoadWithData(), I've created `c` in that way: https://github.com/open-policy-agent/conftest/pull/704/files#diff-796a15f8ba275b5f9972ae2f7353fbf44e0c08d0915fe91b0ba4581c90803663R70